cmdutil/metrics/otel/otel.go

```diff
@@ -0,0 +1,61 @@
+package metrics
+
+import (
+	"context"
+	"net/url"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric"
+	"go.opentelemetry.io/otel/sdk/export/metric"
+
+	"github.com/heroku/x/cmdutil/metrics/honeycomb"
+	"github.com/heroku/x/go-kit/metrics"
+	otelprovider "github.com/heroku/x/go-kit/metrics/provider/otel"
+)
+
+// Config is a reusable configuration struct that contains the necessary
+// environment variables to setup an metrics.Provider
+type Config struct {
+	Enabled             bool     `env:"ENABLE__COLLECTION"`
```

Comment:
silly comment here, but, maybe it's a typo. Why the double underscore?

Reply:
Nice catch! This is a mistake